pkg/catalog/manager/catalog_sync.go

```diff
@@ -18,7 +18,19 @@ func (m *Manager) Sync(key string, obj *v3.Catalog) error {
 		return err
 	}
 
+	// check if user refresh the catalog and reset the retryCount
+	if v3.CatalogConditionRefreshed.IsUnknown(catalog) {
+		catalog.Spec.RetryCount = 0
+	}
+
+	// check retryCount to avoid too many invalid catalog retries
+	catalog.Spec.RetryCount += 1
+	if catalog.Spec.RetryCount > 5 {
+		return nil
+	}
```

Comment:
what's the point of setting retryCount? i thought the framework should follow some exponential backoff time...

Reply:
The catalog controller didn't seem to be stopped by the framework exponential backoff time, not sure it was caused the auto-refresh register, and non-stop reties will causes `too many retires block`.

Reply:
@guangbochen i think we should fix that.....